livekit-ffi/protocol/room.proto

```diff
@@ -407,8 +407,7 @@ message ParticipantMetadataChanged {
 
 message ParticipantAttributesChanged {
   string participant_identity = 1;
-  map<string, string> old_attributes = 2;
-  map<string, string> attributes = 3;
+  map<string, string> changed_attributes = 2;
```

Comment:
if we only send the changed attributes, then each language implementation has to re-construct the currently set attributes from the changed_attributes, right?
Should we maybe just send both, diff + new ones?

Reply:
ahh yes you're right, I keep forgetting that the downstream clients keep their own state (instead of passthrough to rust).

Will update now